.cirrus.yml

```diff
@@ -0,0 +1,75 @@
+test_linux_task:
+  name: Test `patrol develop` on Linux
+  container:
+    image: ghcr.io/cirruslabs/flutter:stable
+    cpu: 6
+    memory: 12G
```

Comment:
that's a lot

Reply:
with 8 it didn't fit in 10 minutes of test building

Reply:
sounds like a `cpu` bottleneck, not `memory`, but ok